Makefile

```
@@ -76,9 +76,9 @@ clean:
 ##@ Development
 
 .PHONY: build
-build: ## Build cilium-policy-viewer
+build: ## Build network-policy-viewer
 	mkdir -p $(BIN_DIR)
-	go build -o $(BIN_DIR)/cilium-policy main.go
+	CGO_ENABLED=0 go build -trimpath -ldflags "-w -s" -o $(BIN_DIR)/npv main.go
```

```suggestion
	go build -trimpath -ldflags "-w -s" -o $(BIN_DIR)/npv main.go
```
nit: go doesn't use cgo by default from 1.20 :)
(Of course, building with `CGO_ENABLED=0` doesn't matter.)
https://tip.golang.org/doc/go1.20#cgo